lib/documents.js

```diff
@@ -26,6 +26,8 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
     // and queries must be supported to find them all...
     // but do not allow multiple `internalId`s for the same `hash` and
     // `externalIdHash`
+    // an index is not created for registration.internalId due to performance
```

Comment:
```suggestion
    // Note: an index is not created for `registration.internalId` because queries
    // based on that field alone are rare; they are not run in hot code paths nor do
    // they need to run quickly -- so the index cost is not justified
```